src/datasources/db/v2/postgres-database.module.spec.ts

```diff
@@ -0,0 +1,73 @@
+import { join } from 'path';
+import { TypeOrmModule } from '@nestjs/typeorm';
+import { Test, type TestingModule } from '@nestjs/testing';
+import { ConfigModule, ConfigService } from '@nestjs/config';
```

Comment:
As before regarding usage of `@/config/configuration.module`.

Reply:
Unfortunately, Since we need `env` variables, it's not possible to use `configuration.module`.